Keen.NET_35/Event.cs

```diff
@@ -25,6 +25,7 @@ public JArray GetSchemas()
                 var client = new RestClient(_serverUrl);
                 var request = new RestRequest("", Method.GET);
                 request.AddHeader("Authorization", _prjSettings.MasterKey);
+                request.AddHeader("Keen-Sdk", KeenUtil.GetSdkVersion());
```

Comment:
I was planning to put "Keen-Sdk", "Authorization", "content-type", "application/json" and so forth in constants in whatever file where we implement an HttpClientFactory in an upcoming change.